quick-start.pod

```diff
@@ -1007,6 +1031,19 @@ happens when we anchor the regular expression to the end of the string:
 In this case, it matches because the last character of the string is an
 exclamation mark.
 
+=head4 Exercise
+
+In this exercise, create the regular expression so that the printed
+statement is true, using character classes and anchors:
+
+    my $string = "Perl was born in 1987";
+    if ($string =~ ) {
+        say "The string ends with a year";
+    }
+    __TEST__
+    like($code, qr/.*$/, 'The expression should use an anchor');
```

Comment:
After having played a little with this test, it turns out that `qr/\$/` matches de dollar in `$string` in the code, so it's not enough. I tried `qr/\$\W/` (dollar followed by non-word) and it seems to work fine. What do you think?